mono/metadata/marshal-ilgen.c

```diff
@@ -4702,7 +4732,7 @@ emit_marshal_vtype_ilgen (EmitMarshalContext *m, int argnum, MonoType *t,
 			static MonoMethod *to_oadate;
 
 			if (!to_oadate)
-				to_oadate = get_method_nofail (date_time_class, "ToOADate", 0, 0);
+				mono_atomic_store_seq (&to_oadate, get_method_nofail (date_time_class, "ToOADate", 0, 0));
```

Comment:
What am I missing? Why isn't this one using `MONO_STATIC_POINTER_INIT`?

Reply:
In some cases they were used across multiple scopes. Let me try again here. Good catch thank you.